backend/coins/eth/etherscan/etherscan.go

```diff
@@ -50,17 +52,22 @@ const ERC20GasErr = "insufficient funds for gas * price + value"
 type EtherScan struct {
 	url        string
 	httpClient *http.Client
+	limiter    rate.Limiter
```

Comment:
usually `*rate.Limiter` so you don't need to dereference `*rate.NewLimiter(...)` below, or is there a reason for making this a value instead of a pointer?